configurations/test/env.yml

```diff
@@ -1,4 +1,25 @@
-AUTH0_DOMAIN: test.auth0.com
```

Comment:
@binh-dam-ibigroup - I've removed the file and the following line from git ignore else it will be included in the next commit:

!configurations/test/env.yml

I've checked datatool-server and it doesn't have this line.

Reply:
... and I've now reverted that update because Travis requires the test env.yml file!

Reply:
Oh that's right.